Enable auto-backfills for devtools framework

airimovici opened the pull request:
This adds support for devtools framework on Sherlock

sarah-clements commented:
hi @airimovici, try rebasing your pr onto latest master to see if it fixes the failing python tests. If that doesn't work, let me know and I'll look into it again tomorrow.

airimovici commented:
Thanks @sarah-clements, I rebased the changes and fixed a linting error and now everything passes.
If everything is ok, I would want to wait until Monday for the merge as I have to monitor the backfill bot's activity for a few days afterwards
